Formula/neovide.rb

```diff
@@ -0,0 +1,25 @@
+class Neovide < Formula
+  desc "No Nonsense Neovim Client in Rust"
+  homepage "https://github.com/neovide/neovide"
+  url "https://github.com/neovide/neovide/archive/tags/0.8.0.tar.gz"
+  sha256 "44822d83c17aebe1fe512c550781ce7699b3135049432d06f1397f9744d05067"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "neovim"
+
+  on_linux do
+    depends_on "fontconfig"
+    depends_on "freetype"
+    depends_on "libxcb"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+    bin.install "target/release/neovide"
+  end
+
+  test do
+    system bin/"neovide", "--remote-tcp=localhost:6666"
```

Comment:
Don't we want to actually start an `nvim` server for this to actually test something? I'm surprised trying to connect to a non-existent server doesn't error, actually.